test/user.test.js

```diff
@@ -2193,6 +2193,72 @@ describe('User', function() {
     });
   });
 
+  describe('Verification after updating email', function() {
+    var nonVerifiedUser;
+    var originalMyEmailCred = { email: 'original@example.com', password: 'bar',
+    emailVerified: true };
+    var myUpdatedCred = { email: 'updated@example.com' };
+    it('sets verification to false after email update if verification is required',
+    function(done) {
+      User.settings.emailVerificationRequired = true;
+      async.series([
+        function createUser(next) {
+          User.create(originalMyEmailCred, function(err, specialInstance) {
```

Comment:
AFAICT, this setup is shared by both tests. Please extract it into a shared `beforeEach` hook.

```js
beforeEach(setupTestUser);

it(...);

it(...);

function setupTestUser(done) {
  User.create(originalMyEmailCred, function(err, inst) {
    if (err) return done(err);
    nonVerifiedUser = inst; // use a different name than nonVerifiedUser
    done();
  });
}
```